src/main/resources/assets/groovyscript/lang/en_us.lang

```diff
@@ -621,6 +622,47 @@ groovyscript.wiki.botania.rune_altar.input.required=that `input` IIngredients ca
 groovyscript.wiki.botania.rune_altar.mana.value=Sets the mana cost of processing the recipe
 
 
+# Tinkers
+groovyscript.wiki.tconstruct.smeltery_fuel.title=Smeltery Fuel
+groovyscript.wiki.tconstruct.smeltery_fuel.description=Add or remove smeltery fuel types
```

Comment:
could you better describe what is going on in these descriptions? eg `Modifies what fluids are accepted as fuels for the Smeltery and how long each fuels the Smeltery.`